spec/features/integration_spec.rb

```diff
@@ -1,7 +1,7 @@
 require 'rails_helper'
 
-describe "an exhibit" do
-  it "loads the home page" do
-    visit "/"
+describe 'an exhibit' do
+  it 'loads the home page' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.